src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -117,7 +124,8 @@ public void save(AddressBook addressBook) throws StorageOperationException {
      * @throws StorageOperationException if there were errors reading and/or converting data from file.
      */
     public AddressBook load() throws StorageOperationException {
-        try (final Reader fileReader =
+    	isDeleted();
```

Comment:
when checking on load, wouldn't the program throw the exception when running for the very first time?